framework/packages/abstract-interface/src/account/mod.rs

```diff
@@ -177,6 +177,17 @@ impl<Chain: CwEnv> AbstractAccount<Chain> {
         self.install_module_parse_addr(app, Some(&custom_init_msg), funds)
     }
 
+    /// Installs an standalone from an standalone object
+    pub fn install_standalone<CustomInitMsg: Serialize, T: ContractInstance<Chain>>(
```

Comment:
add `install_standalone_witd_dependencies`. 

Also see https://discord.com/channels/976930997654405132/976931523582382090/1248567756740100146

Reply:
> add `install_standalone_witd_dependencies`.

Done
 
> Also see https://discord.com/channels/976930997654405132/976931523582382090/1248567756740100146

Let's have it in separate PR. It's easy to implement if we just check if it's installed and "pop" from the list if it is. But I'm not sure what we do with version miss-matches? Do we just let it fail or try to update if it's newer version?